### DISCUSSION TITLE
support RTL

### DISCUSSION THREAD

**Discussion by Mostafa-Moafi** [dnnsoftware/Dnn.Platform#5289] [Ideas]
Is it possible to provide RTL support in future features?

I can help to implement this feature.

do you agree?

**Comment by valadas** [comment]
I am no expert in this area, but sure

**Comment by Mostafa-Moafi** [comment]
I know will be implemented RTL in DNN and I implemented this feature. (Almost)

But I like this feature in the main DNN.



![1](https://user-images.githubusercontent.com/58891213/189578048-956a5162-162f-4564-b5d0-fce3d7952ca2.png)

As shown in the above picture implemented RTL and Persian language in DNN.

**Comment by valadas** [comment]
If you mean the Html editor, I am pretty sure this is already supported:
![image](https://user-images.githubusercontent.com/6371568/189693116-3427dd0d-0023-4a23-9a68-0a3230d5c2f1.png)

You should find this by going to Site Settings => Behaviour => HTML Editor Manager

**Reply by Mostafa-Moafi** [reply]
No, I mean PersonaBar

**Reply by valadas** [reply]
Oh, yeah, in that area I have no clue. PRs are welcome though
